internal/action/delete.go

```diff
@@ -107,5 +108,5 @@ func (s *Action) deleteKeyFromYAML(ctx context.Context, name, key string) error
 		return exit.Error(exit.IO, err, "Can not delete key %q from %q: %s", key, name, err)
 	}
 
-	return nil
+	return hook.Invoke(ctx, "delete.post-rm", key+"@"+name)
```

Comment:
Is the `@` symbol really not authorised in keys currently?

I tested it and it seems a key can contain the symbol "@" right now, so this might not be a great separator, since people might be using email addresses as keys.

Reply:
Yeah, you're right. How should we signal to the hook that this is a key? Maybe just as a separate arg?!

Reply:
I feel a separate argument would be best, yes.